src/helpers/shareHelper.js

```diff
@@ -38,20 +39,26 @@ export const openShare = async ({ message, title, url }) => {
 };
 
 export const shareMessage = (data, query) => {
+  const urls = mergeWebUrls({
+    webUrls: data.webUrls,
+    contact: data.contact,
+    contacts: data.contacts
+  })?.map(({ url }) => `\n${url}`);
```

Comment:
nice for finding and using `mergeWebUrls` here!
i am missing the "Link: " part in front of every link, that was requested in the ticket comments.

Reply:
I thought there might be more than one Link. At the moment I have added the `Link` text at the beginning of each link. If you want, I can make the `Links` write in case there is more than one link with more control.
https://github.com/smart-village-solutions/smart-village-app-app/pull/916/commits/081c27876bf3cbd07a86a776fd5aff7f5aef0b28

Reply:
I do not think that something with "Links" is necessary as "Link: ..." was requested. Each link can have its "Link: ".